### PR TITLE
server: Add support of Yaml validation request #221

### PR DESCRIPTION
**Pull request by akryvko**
#221

### CODE REVIEW

File: server/src/test/resources/fixtures/api_violations_request.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            description: User is not authenticated
+          403:
+            description: User is not authorized
+          404:
+            description: Identifier is not found
+            schema:
+              $ref: '#definitions/Problem'
+        security:
+          - oauth2:
+            - 'cross-device-graph-service.read'
```
Comment (netme) [inline server/src/test/resources/fixtures/api_violations_request.yaml]: Optional: could you please remove non-necessary empty lines from this file?

**Comment by netme** [comment]
👍

File: server/src/test/resources/fixtures/api_violations_request.yaml (hunk 1 above)
Comment (akryvko) [inline server/src/test/resources/fixtures/api_violations_request.yaml]: they are added intentionally for line-numbers feature testing. Let's keep them for a while

**Comment by maxim-tschumak** [comment]
:+1:

**Pull request merged**